defi/src/api2/routes/index.ts

```diff
@@ -447,6 +447,12 @@ async function getDimensionsMetadataRoute(_req: HyperExpress.Request, res: Hyper
   return successResponse(res, await getDimensionsMetadata(), 60);
 }
 
+export function setProRoutes(router: HyperExpress.Router, _routerBasePath: string) {
+  router.get("/summaryBreakdown/:type/:name", ew(getDimensionBreakdownProtocolFileRoute))
+  router.get("/aggregates/:type/:name", ew(getAggregatesProtocolFileRoute))
+  router.get("/aggregatesBreakdown/:type/:name", ew(getAggregatesBreakdownProtocolFileRoute))
+}
+
```

Comment:
can we combine these two to `/v2/metrics/:type/:name?breakdown=true|false`
adding versioning to make it easy for us to deprecate api in the future, and breakdown as request parameter to cut down on the number of exposed routes